django_ninja_crudl/crudl.py

```
@@ -30,381 +33,114 @@
+                        status.HTTP_404_NOT_FOUND: Error404NotFoundSchema,
+                        status.HTTP_409_CONFLICT: Error409ConflictSchema,
+                        status.HTTP_422_UNPROCESSABLE_ENTITY: Error422UnprocessableEntitySchema,
+                        status.HTTP_503_SERVICE_UNAVAILABLE: Error503ServiceUnavailableSchema,
                     },
                     openapi_extra=create_schema_extra,
                     by_alias=True,
                 )
                 @transaction.atomic
-                @add_function_arguments(create_path)
+                @add_function_arguments(config.create_path)
                 def create(
```

Ruff (0.8.2)

`create` is too complex (14 > 10)

(C901)